core/templates/pages/creator-dashboard-page/creator-dashboard-page.component.html

```diff
@@ -10,7 +10,7 @@ <h1 tabindex="0" [innerHTML]="'I18N_DASHBOARD_CREATOR_DASHBOARD' | translate"></
        *ngIf="explorationsList?.length > 0 || collectionsList?.length > 0"
        [ngClass]="{'oppia-creator-dashboard-mobile-mode': checkMobileView()}">
     <mat-card layout="row" class="oppia-mat-card">
-      <div class="average-ratings stats-card">
+      <div class="average-ratings stats-card" [attr.aria-label]="'Average ratings ' + (dashboardStats.averageRatings || 'N/A')" tabindex="0" >
```

Comment:
```suggestion
      <div class="average-ratings stats-card"
           [attr.aria-label]="'Average ratings ' + (dashboardStats.averageRatings || 'N/A')"
           tabindex="0" >
```
Ditto here and elsewhere

Reply:
Done